src/julia/tools.py

```diff
@@ -139,7 +139,9 @@ def julia_py_executable(executable=sys.executable):
     """
     Path to ``julia-py`` executable installed for this Python executable.
     """
-    stempath = os.path.join(os.path.dirname(executable), "julia-py")
+    import sysconfig
```

Comment:
Can you move this up to the beginning of the file?

There are already imports of many packages up there so I don't think we need to lazy-import this particular package.

Reply:
Yes, and I also remove the default argument. It shall never get used.